framework/src/main/java/org/apache/drill/test/framework/Utils.java

```diff
@@ -169,7 +170,21 @@ public static ConnectionPool getConnectionPool() {
         boolean foundTests = false;
         for (String testGroup : testGroups) {
           if (categories != null && !categories.contains(testGroup)) {
-            continue;
+            if (categories.contains("smoke") &&
```

Comment:
I think there is a difference between functional_maprdb and simba_jdbc.  Most tests should be run with simba_jdbc.  Most tests should not run with functional_maprdb.  I think the majority of the test suites should run with simba_jdbc.  So why do this manually?  Why not have the test framework handle this for us?